climpred/bootstrap.py

```diff
@@ -1,11 +1,103 @@
 import numpy as np
 import xarray as xr
 
-from .prediction import compute_perfect_model, compute_persistence_pm
+from .metrics import POSITIVELY_ORIENTED_METRICS
+from .prediction import (compute_hindcast, compute_perfect_model,
+                         compute_persistence)
 from .stats import DPP, xr_varweighted_mean_period
 
 
-def pseudo_ens(ds, control):
+def _distribution_to_ci(ds, ci_low, ci_high, dim='bootstrap'):
+    """Get confidence intervals from bootstrapped distribution.
+
+    Needed for bootstrapping confidence intervals and p_values of a metric.
+
+    Args:
+        ds (xarray object): distribution.
+        ci_low (float): low confidence interval.
+        ci_high (float): high confidence interval.
+        dim (str): dimension to apply xr.quantile to. Default: 'bootstrap'
+
+    Returns:
+        uninit_hind (xarray object): uninitialize hindcast with hind.coords.
+    """
+    ## TODO: get rif od try, except logic
+    try:  # incase of lazy results compute
+        if len(ds.chunks) >= 1:
+            ds = ds.compute()
+    except:
+        pass
+    ds_ci = ds.quantile(q=[ci_low, ci_high], dim=dim)
+    return ds_ci
+
+
+def _pvalue_from_distributions(simple_fct, init, metric='pearson_r'):
+    """Get probability that skill of simple_fct is larger than
```

Comment:
Should change `simple_fct` to "a simple forecast (e.g., persistence or uninitialized skill).